packages/orchestration/src/exos/zcf-for-flow.js

```diff
@@ -0,0 +1,75 @@
+import { VowShape } from '@agoric/vow';
+
+/**
+ * @import {HostOf} from '@agoric/async-flow';
+ * @import {VowTools} from '@agoric/vow';
+ * @import {Zone} from '@agoric/zone';
+ */
+
+import { M } from '@endo/patterns';
+
+export const ZcfI = M.interface(
```

Comment:
this is different than `ZcfI` exported from `@agoric/zoe`. Let's have a distinct name.

I'd suggest `ZcfForFlowsI` but that's quite clunky. But that goes for the whole Exo name imo.

Would `FlowsZcf`, `FlowsZcfI` and `flowsZcf` suffice? JSdoc could resolve any ambiguity

Reply:
sorting with ZCF seems worthwhile. going with `ZcfForFlowsI`